vector/src/main/java/im/vector/app/features/voicebroadcast/usecase/StopVoiceBroadcastUseCase.kt

```diff
@@ -53,14 +56,20 @@ class StopVoiceBroadcastUseCase @Inject constructor(
     private suspend fun stopVoiceBroadcast(room: Room, reference: RelationDefaultContent?) {
         Timber.d("## StopVoiceBroadcastUseCase: Send new voice broadcast info state event")
         room.stateService().sendStateEvent(
-                eventType = STATE_ROOM_VOICE_BROADCAST_INFO,
+                eventType = VoiceBroadcastConstants.STATE_ROOM_VOICE_BROADCAST_INFO,
                 stateKey = session.myUserId,
                 body = MessageVoiceBroadcastInfoContent(
                         relatesTo = reference,
                         voiceBroadcastStateStr = VoiceBroadcastState.STOPPED.value,
                 ).toContent(),
         )
 
-        // TODO stop recording audio files
+        stopRecording()
+    }
+
+    private fun stopRecording() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
I think this check could be removed.

Reply:
https://github.com/vector-im/element-android/pull/7363/commits/400118ed3ec8eb19bb4ba6011f5b3f5c6c0816b7